includes/class-sensei-lesson.php

```diff
@@ -4726,9 +4731,14 @@ public static function user_not_taking_course_message() {
 	 */
 	public static function course_signup_link() {
 
+		$lesson_id = get_the_ID();
+		if ( $lesson_id ) {
```

Comment:
Probably a typo here
```suggestion
		if ( ! $lesson_id ) {
```